test/python/scenarios.json

```diff
@@ -1,4 +1,13 @@
 {
+    "install_python_setuptools_vulnerability": {
+        "image": "debian:bookworm",
```

Comment:
Once you address previous comment, can you also add a new test scenario with image `python:3:11` ?

Reply:
Done in PR # 815